dev/release-aux/openssl-announce-pre-release.tmpl

```diff
@@ -0,0 +1,49 @@
+
+   OpenSSL version $release_text released
+   ======================================
+
+   OpenSSL - The Open Source toolkit for SSL/TLS
+   https://www.openssl.org/
+
+   OpenSSL $series is currently in $label.
+
+   OpenSSL $release_text has now been made available.  For details of
+   changes and known issues see the release notes at:
```

Comment:
Adding a link to the wiki notes would be a useful addition. Either in this PR or a subsequent one.

Reply:
Other PR